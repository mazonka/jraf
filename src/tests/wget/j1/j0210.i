command=jr read 0 /aaa