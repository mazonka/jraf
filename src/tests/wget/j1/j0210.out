JRAF_ERR bad path /aaa